### PR TITLE
getUI

### PR DESCRIPTION
**Pull request by ahumphreys87**
Cherry picked 10e2d77 onto minor - opened in favour of #2115 

Amended docs to use `this.getUI` over `this.ui`

### CODE REVIEW

**Comment by ahumphreys87** [comment]
ping @marionettejs/marionette-core 

do we need any more docs than this?

**Comment by samccone** [comment]
yeah I think we need some docs in the view.md file and also some docs in the view.yml

File: docs/marionette.view.md [hunk 1]
```diff
@@ -412,7 +412,7 @@ or other ui element that you wish to set a css class to it.
 Instead of having jQuery selectors hanging around in the view's code
 you can define a `ui` hash that contains a mapping between the
 ui element's name and its jQuery selector. Afterwards you can simply
-access it via `this.ui.elementName`.
```
Comment (jasonLaster) [inline docs/marionette.view.md]: lets also add a `getUI` section to the docs

**Comment by jasonLaster** [comment]
Thanks @ahumphreys87

**Comment by ahumphreys87** [comment]
:boom: updated

**Comment by ahumphreys87** [comment]
:blush: fixed trailing space error

**Comment by jasonLaster** [comment]
+1 docs look great

**Comment by jmeas** [comment]
:1234:

**Comment by jmeas** [comment]
:eight_pointed_black_star:

**Comment by jmeas** [comment]
💔

**Comment by jasonLaster** [comment]
Jmeas you clown. You drinking and commenting?

**Comment by jdaudier** [comment]
:joy:

**Comment by jmeas** [comment]
yes

**Comment by jmeas** [comment]
on a plane

**Comment by thejameskyle** [comment]
Drunk @jmeas on a plane is the best @jmeas

**Comment by jmeas** [comment]
omg I'm almost there

**Comment by jdaudier** [comment]
Where?

**Comment by jmeas** [comment]
Tampa!!!!!

**Comment by jmeas** [comment]
ok I made it. see you folks on Tuesday when I fly back to boston

**Pull request merged**